src/write.ts

```diff
@@ -566,8 +580,10 @@ async function handleSummary(benchName: string, currBench: Benchmark, prevBench:
 
 function getRatio(tool: ToolType, prev: BenchmarkResult, current: BenchmarkResult) {
     if (prev.value === 0 && current.value === 0) return 1;
+    const prevMs = toMs(prev.value, prev.unit);
+    const currentMs = toMs(current.value, current.unit);
```

Comment:
Not all benchmarks are output results in units of time and this logic will apply to all of them which would basically output 0 for all benchmarks that don't use `us`, `ms` or `s` as unit